site/docs/samples/notebook-read.md

```diff
@@ -72,54 +78,72 @@ stringData:
 EOF
 ```
 
-Then, register the data asset itself in the data catalog `katalog` used for samples. Replace the values for `endpoint`, `bucket` and `object_key` with values from the object storage service that you used and run:
-
-```yaml
-cat << EOF | kubectl apply -f -
-apiVersion: katalog.fybrik.io/v1alpha1
-kind: Asset
-metadata:
-  name: paysim-csv
-spec:
-  secretRef: 
-    name: paysim-csv
-  details:
-    dataFormat: csv
-    connection:
-      name: s3
-      s3:
-        endpoint: "http://localstack.fybrik-notebook-sample.svc.cluster.local:4566"
-        bucket: "demo"
-        object_key: "PS_20174392719_1491204439457_log.csv"
-  metadata:
-    name: Synthetic Financial Datasets For Fraud Detection
-    geography: theshire 
-    tags:
-      finance: true
-    columns:
-      - name: nameOrig
-        tags:
-          PII: true
-      - name: oldbalanceOrg
-        tags:
-          PII: true
-      - name: newbalanceOrig
-        tags:
-          PII: true
+Next, register the data asset itself in the data catalog.
+We use port-forwarding to send asset creation requests to the OpenMetadata connector.
+```bash
+kubectl port-forward svc/openmetadata-connector -n fybrik-system 8081:8080 &
+cat << EOF | curl -X POST localhost:8081/createAsset -d @-
+{
+  "destinationCatalogID": "openmetadata",
+  "destinationAssetID": "paysim-csv",
+  "credentials": "/v1/kubernetes-secrets/paysim-csv?namespace=fybrik-notebook-sample",
+  "details": {
+    "dataFormat": "csv",
+    "connection": {
+      "name": "s3",
+      "s3": {
+        "endpoint": "http://localstack.fybrik-notebook-sample.svc.cluster.local:4566",
+        "bucket": "demo",
+        "object_key": "PS_20174392719_1491204439457_log.csv"
+      }
+    }
+  },
+  "resourceMetadata": {
+    "name": "Synthetic Financial Datasets For Fraud Detection",
+    "geography": "theshire ",
+    "tags": {
+      "finance": "true"
+    },
+    "columns": [
+      {
+        "name": "nameOrig",
+        "tags": {
+          "PII": "true"
+        }
+      },
+      {
+        "name": "oldbalanceOrg",
+        "tags": {
+          "PII": "true"
+        }
+      },
+      {
+        "name": "newbalanceOrig",
+        "tags": {
+          "PII": "true"
+        }
+      }
+    ]
+  }
+}
```

Comment:
I think very important to provide instructions how to register an asset in OMD by using its GUI.

Reply:
I will provide the instructions in a separate PR